v1.0/v1.0/default_path_job.yml

```diff
@@ -0,0 +1,3 @@
+file1:
+  class: File
+  path: hello.txt
```

Comment:
A lot of these paths seemed to be switched to locations in these test cases but in light of this test case I'm wondering if the "may" in the following sentence should be a "must"?

> As a special case, if the path field is provided but the location field is not, an implementation may assign the value of the path field to location, and remove the path field.